reqs/management/commands/import_reqs.py

```diff
@@ -251,6 +271,7 @@ def add(self, row):
             req_id=req_id, defaults=params)
         self.connections.extend(self.keywords.connections(row, req.pk))
         self.req_ids.add(req_id)
+        self.last_id = req_id
```

Comment:
It makes more sense to me to set it only after it's been successfully inserted, as with adding to `self.req_ids`.